ethcore/verification/src/queue/kind.rs

```diff
@@ -96,7 +96,7 @@ pub mod blocks {
 	impl Kind for Blocks {
 		type Input = Unverified;
 		type Unverified = Unverified;
-		type Verified = PreverifiedBlockWithBytes;
+		type Verified = (PreverifiedBlock, Vec<u8>);
```

Comment:
1. I've replaced `PreverifiedBlockWithBytes` with a tuple in the last commit, but not sure if it improved readability.
2. Using `Vec<u8>` here instead of `Bytes` not to introduce another dependency just for a type alias.

Reply:
Maybe add a comment here explaining what the `Vec<u8>` contains: the RLP representation of the block?